cmd/template/framework/files/air.toml.tmpl

```diff
@@ -5,7 +5,7 @@ tmp_dir = "tmp"
 [build]
   args_bin = []
   bin = "./tmp/main"
-  cmd = "make run"
+  cmd = "make build"
```

Comment:
`cmd = "go build -o ./tmp/main ./cmd/api"`

we can run directly like this instead using build command. imo making build file in tmp folder not a good practice